bucket/virt-viewer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "10.0-1.0",
+    "description": "Open source tool to connect to virtual machines to display VNC or SPICE",
+    "homepage": "https://virt-manager.org/download/",
```

Comment:
Why download/??

Reply:
I was not sure what to put here. I will change it to https://virt-manager.org